second-analysis-steps/building-decays-part2.md

```diff
@@ -31,15 +32,17 @@ Note how we needed to use the `CombineParticles` from `GaudiConfUtils.Configurab
 This is because the LHCb algorithms are configured as singletons and it is mandatory to give them a name, which we don't want to in `SimpleSelection` (we want to skip steps!).
 
 {% callout "Why `ConfigurableGenerators`?" %}
+
 If we had tried to simply use `CombineParticles` inside our 
 `SimpleSelection`, we would have seen it fail with the following error
 
-```output
+```
 NameError: Could not instantiate Selection because input Configurable CombineParticles has default name. This is too unsafe to be allowed.
 ```
```

Comment:
Can we avoid color-highlighting here? This line is treated as code currently.

Reply:
Fixed. I hadn't realised the default syntax highlighting is `python3` and not `none`: https://www.sphinx-doc.org/en/1.5/config.html#confval-highlight_language